autoware_adapi_v1_msgs/system/msg/Heartbeat.msg

```diff
@@ -0,0 +1 @@
+builtin_interfaces/Time stamp
```

Comment:
Do you not need a counter?

Reply:
@yukkysaito I've also suggested the same idea [above](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/50#issuecomment-2022460258)

Reply:
Thank you. Add a message counter https://github.com/autowarefoundation/autoware_adapi_msgs/pull/50/commits/cd39fdd0260c3b2e3090a785b411fe1c99302725.